### PR TITLE
[CI:DOCS] rootless_tutorial: modernize

### PR DESCRIPTION
**Pull request by edsantiago**
- We can assume that cgroups v2 and rootless overlayfs are the
  default everywhere.

- Remove RHEL7-only instructions

- add clear '$' and '#' prompts to rootless and root commands

- other minor consistency cleanups

Ref: #20669

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
Update rootless tutorial for relevance in a cgroups-v2 and overlayfs-default world
```

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
This was way out of date.

File: docs/tutorials/rootless_tutorial.md [hunk 1]
```diff
@@ -142,16 +106,16 @@ The three main configuration files are [containers.conf](https://github.com/cont
 Podman reads
 1. `/usr/share/containers/containers.conf`
 2. `/etc/containers/containers.conf`
-3. `$HOME/.config/containers/containers.conf`
+3. `${XDG_CONFIG_HOME}/containers.conf`
```
Comment (Luap99) [inline docs/tutorials/rootless_tutorial.md:109]: 1. path is of, it must be `${XDG_CONFIG_HOME}/containers/containers.conf`
 2. XDG_CONFIG_HOME is not set by default so it should say that it uses $HOME/.config if it is unset IMO.

File: docs/tutorials/rootless_tutorial.md [hunk 2]
```diff
@@ -142,16 +106,16 @@ The three main configuration files are [containers.conf](https://github.com/cont
...
 
-if they exist in that order. Each file can override the previous for particular fields.
+if they exist, in that order. Each file can override the previous for particular fields.
 
 #### storage.conf
 For `storage.conf` the order is
 1. `/etc/containers/storage.conf`
-2. `$HOME/.config/containers/storage.conf`
+2. `${XDG_CONFIG_HOME}/storage.conf`
```
Comment (Luap99) [inline docs/tutorials/rootless_tutorial.md:116]: same here

**Comment by baude** [comment]
thanks @edsantiago 

/lgtm

File: docs/tutorials/rootless_tutorial.md [hunk 3]
```diff
@@ -163,21 +127,21 @@ runroot=""
 ```
 In rootless Podman these fields default to
 ```
-graphroot="$HOME/.local/share/containers/storage"
-runroot="$XDG_RUNTIME_DIR/containers"
+graphroot="${XDG_DATA_HOME}/containers/storage"
```
Comment (Luap99) [inline docs/tutorials/rootless_tutorial.md:130]: also here XDG_DATA_HOME is not set by default so it should state the default used.

File: docs/tutorials/rootless_tutorial.md [hunk 4]
```diff
@@ -163,21 +127,21 @@ runroot=""
...
 ```
 [$XDG_RUNTIME_DIR](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables) defaults on most systems to `/run/user/$UID`.
 
 #### registries
-Registry configuration is read in by this order
+Registry configuration is read in this order
 1. `/etc/containers/registries.conf`
 2. `/etc/containers/registries.d/*`
-3. `HOME/.config/containers/registries.conf`
+3. `${XDG_CONFIG_HOME}/registries.conf`
```
Comment (Luap99) [inline docs/tutorials/rootless_tutorial.md:139]: also here path is wrong

**Review by Luap99 — CHANGES_REQUESTED** [review]

**Comment by edsantiago** [comment]
Thank you @luap99, I will open a new PR

**Pull request merged**